domain/fqdn.go

```diff
@@ -1,8 +1,25 @@
 package domain
 
+import (
+	"encoding/json"
+
+	model "github.com/owasp-amass/open-asset-model"
```

Comment:
maybe `types` could be a better alias 

```suggestion
	types "github.com/owasp-amass/open-asset-model"
```